spec/ddtrace/contrib/rails/rails_log_auto_injection_spec.rb

```diff
@@ -0,0 +1,151 @@
+require 'ddtrace/contrib/rails/rails_helper'
+
+RSpec.describe 'Rails Rack Rum Injection middleware' do
```

Comment:
Is Rum related to log injection? I'm not sure I understand the overlap.

Reply:
ah, leaked over, good catch will updated